projects/ngrx.io/content/navigation.json

```diff
@@ -114,6 +114,10 @@
                 {
                   "title": "Feature Creators",
                   "url": "guide/store/feature-creators"
+                },
+                {
+                  "title": "Action Groups",
```

Comment:
Not related to this PR, but are feature creators and action groups "advanced"?
Should we classify them as "more developer friendly" ?

Reply:
let me know if we want to move these sections, took care of rest of the comments.

Reply:
To keep this PR focused, I would leave it as is.
We can make the change when this PR is merged.